scripts/create_project.sh

```diff
@@ -5,16 +5,16 @@ ACCOUNT_ID=$2
 
 ROLE_NAME=CelloSampleRole
 
-if [ -z $ARGO_CLOUDOPS_PROJECT_NAME ]; then
-    export ARGO_CLOUDOPS_PROJECT_NAME=project1
+if [ -z $CELLO_PROJECT_NAME ]; then
+    export CELLO_PROJECT_NAME=project1
 fi
 
-if [ -z $ARGO_CLOUDOPS_TARGET_NAME ]; then
-    export ARGO_CLOUDOPS_TARGET_NAME=target1
+if [ -z $CELLO_TARGET_NAME ]; then
+    export CELLO_TARGET_NAME=target1
 fi
 
-if [ -z $ARGO_CLOUDOPS_SERVICE_ADDR ]; then
-    export ARGO_CLOUDOPS_SERVICE_ADDR=https://localhost:8443
+if [ -z $CELLO_SERVICE_ADDR ]; then
```

Comment:
I think this needs to be `ARGO_CLOUDOPS` still? I think this one is only being used by the cli